sponsor-dapp-v2/src/views/ManagePositions.js

```diff
@@ -246,12 +249,14 @@ function ManagePositions(props) {
 
                           <td>
                             <strong>
-                              {format(data.totalHoldings)} DAI ({data.collateralizationRatio}%)
+                              {format(data.totalHoldings)} DAI ({data.collateralizationRatio.toString()}%)
```

Comment:
(Just for my own curiosity) Does the `{}` syntax not stringify it for you? What was it showing previously?

Reply:
The BN wasn't getting displayed properly. Though this isn't fully correct either: we probably need to format the percents better/more consistently.